fiftyone/utils/labelstudio.py

```diff
@@ -624,7 +628,7 @@ def _update_dict(src_dict, update_dict):
     return new
 
 
-def export_label_to_label_studio(label, full_result=None):
+def export_label_to_label_studio(sample, label_field, label_type, full_result=None):
```

Comment:
`full_result` already includes `original_width` and `original_height`, so can you just use that and avoid changing the signature of this method?